app/views/patients/_form.html.erb

```diff
@@ -15,81 +15,87 @@
             <div class="form-group">
                 <%= f.label :last_name %> <%= f.text_field :last_name, class: "form-control text-capitalize"  %>
             </div>
         </div>
     </div>
 
     <div class="form-row">
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :date_of_birth %> <%= f.date_field :date_of_birth, class: "form-control" %>
             </div>
         </div>
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :permanent_district %>
                     <div id="district-div">
                         <%= f.text_field :permanent_district, class: "form-control typeahead" %>
-                    </div>                
-                
+                    </div>
+
             </div>
         </div>
     </div>
 
     <div class="form-row">
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :fathers_name %> <%= f.text_field :fathers_name, class: "form-control text-capitalize" %>
             </div>
         </div>
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :fathers_occupation %> <%= f.text_field :fathers_occupation, class: "form-control text-capitalize" %>
             </div>
         </div>
     </div>
     <div class="form-row">
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :mothers_name %> <%= f.text_field :mothers_name, class: "form-control text-capitalize" %>
             </div>
         </div>
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :mothers_occupation %> <%= f.text_field :mothers_occupation, class: "form-control text-capitalize" %>
             </div>
         </div>
     </div>
 
     <div class="form-check">
-        <%= f.label :present_permanent_same, class: 'form-check-label' do %> 
+        <%= f.label :present_permanent_same, class: 'form-check-label' do %>
             <%= f.check_box :present_permanent_same, id: 'pp_same', class: 'form-check-input' %>
             Present address is same as permanent address
         <% end %>
     </div>
 
     <div class="form-row">
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :permanent_address %> <%= f.text_area :permanent_address, class: "form-control text-capitalize" %>
             </div>
         </div>
-        <div class="col">    
+        <div class="col">
             <div class="form-group">
                 <%= f.label :present_address %> <%= f.text_area :present_address, id: 'present_address', class: "form-control text-capitalize"%>
             </div>
         </div>
     </div>
-    
+
+    <div class="form-row">
```

Comment:
can you please add screenshots of what the form and the patients page looks like now?